xml/System.Data.Common/DbCommand.xml

```diff
@@ -1210,8 +1211,8 @@
         <Parameter Name="behavior" Type="System.Data.CommandBehavior" Index="0" FrameworkAlternate="dotnet-uwp-10.0;net-5.0;net-6.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8;netstandard-2.0;netstandard-2.1;xamarinandroid-7.1;xamarinios-10.8;xamarinmac-3.0" />
       </Parameters>
       <Docs>
-        <param name="behavior">One of the enumeration values that specified the command behavior.</param>
-        <summary>An asynchronous version of <see cref="Overload:System.Data.Common.DbCommand.ExecuteReader" />, which executes the <see cref="P:System.Data.Common.DbCommand.CommandText" /> against the <see cref="P:System.Data.Common.DbCommand.Connection" /> and returns a <see cref="T:System.Data.Common.DbDataReader" />.  
+        <param name="behavior">An instance of <see cref="T:System.Data.CommandBehavior" />, specifying options for command execution and data retrieval.</param>
```

Comment:
The original wording seems better here, according to the [guidelines](https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors). Also, the type of parameters and return values is automatically displayed, so we're advised not to include it again in the description.

```suggestion
        <param name="behavior">One of the enumeration values that specifies how the command should execute and how data should be retrieved.</param>
```